javascript/react-props_greeting/src/App.js

```diff
@@ -1,5 +1,23 @@
 import "./styles.css";
 
 export default function App() {
-  return <h1>Replace me with your component!</h1>;
+  return <Greeting name="Marcell" />;
 }
+function Greeting({ name }) {
+  if (name === "Marcell") {
+    return <h1>Hello Coach !</h1>;
+  }
+  return <h1>Hello {name} !</h1>;
```

Comment:
... and incase this does actually work with the if-statement, then I think you're missing an { here before the second return-statement.